app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+#
+class User < ActiveRecord::Base
+  has_secure_password
+  has_one :profile
+  has_many :collections
+
+  validates :name, presence: true
```

Comment:
Wouldn't this bork all kinds of things since names aren't unique? Like if both myself & my grandfather attempted to sign up there would be an error since our names are the same.